new_samples/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"github.com/uber-common/cadence-samples/new_samples/worker"
+	"github.com/uber-common/cadence-samples/new_samples/workflows"
+	"go.uber.org/cadence/activity"
+	"go.uber.org/cadence/workflow"
+	"log"
+	"net/http"
+)
+
+func init() {
+	// HelloWorld workflow
+	workflow.RegisterWithOptions(workflows.HelloWorldWorkflow, workflow.RegisterOptions{Name: "cadence_samples.HelloWorldWorkflow"})
+	activity.RegisterWithOptions(workflows.HelloWorldActivity, activity.RegisterOptions{Name: "cadence_samples.HelloWorldActivity"})
```

Comment:
move to the worker package probably, and register on the worker rather than globally.  these are deprecated.

Reply:
done. yeah it looks like this function is deprecated

Reply:
yea.  if it's not appearing in your IDE, we might want to see if we can fix that.

Goland is pretty permissive and definitely displays this, but vscode seems suuuuuper picky.  Others might be as well, as vscode pretty much just uses gopls.